http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -762,15 +752,16 @@ class ClusterMonitorActor(
 
   private def recordStatusTransitionMetrics(clusterUI: RuntimeUI,
                                             origStatus: RuntimeStatus,
-                                            finalStatus: RuntimeStatus): IO[Unit] =
+                                            finalStatus: RuntimeStatus,
+                                            cloudService: CloudService): IO[Unit] =
     for {
       endTime <- IO(System.currentTimeMillis)
-      baseName = s"ClusterMonitor/${clusterUI.asString}/${origStatus}->${finalStatus}"
-      counterName = s"${baseName}/count"
-      timerName = s"${baseName}/timer"
+      metricsName = s"monitor/${clusterUI.asString}/transition/${origStatus}_to_${finalStatus}"
```

Comment:
Just curious - we used slash-separated names because they were wildcard-able in NewRelic.. are there different metric naming conventions for Stackdriver?

Also I was thinking it would be good to have a reference page with all our metric names.

Reply:
it doesn't work the way newrelic does...we can group more flexibly with "tags" opencensus term (label in stackdriver). Using `\` is more of just naming convention thing.

I think reference page can grow out of sync pretty easily and tedious to maintain. All leo custom metrics starts with `OpenCensus/leonardo/xxx`, and if you type `leonardo`, you'll see all of them.  Maybe I can leave general comment about how metrics names will look like